build_debian.sh

```diff
@@ -499,8 +504,23 @@ fi
 ## Remove gcc and python dev pkgs
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove gcc libpython2.7-dev
 
+if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install u-boot-tools mtd-utils
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-mark manual u-boot-tools mtd-utils
```

Comment:
I think we can installer them in x86 as well.

Reply:
No, we cant. these packages are specific to arch other than x86/amd

Reply:
i do not understand here. https://packages.debian.org/search?keywords=mtd-utils

Reply:
same for the u-boot-tools. https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=u-boot-tools

Reply:
will update.